### ISSUE TITLE
Latest AdoptJDK release is missing

### ISSUE THREAD

**Issue by normanmaurer** [shyiko/jabba#767]
See title

**Comment by jponge** [comment]
Looks like maintainers are busy, there hasn't been any update over the last 3 months.

**Comment by willcohen** [comment]
Sorry for the delay. Autodiscovery (https://travis-ci.org/github/shyiko/jabba-autodiscovery/builds/759293806) has been broken since Zulu added version numbers with four decimal point sections, and I’m rusty enough with regexps that every time I try to get it working I get frustrated half way through. I’ll try to get my in-progress in a branch there shortly, but once that gets unstuck the auto updates to jdks should be back on track.

**Comment by jponge** [comment]
I think you can close this issue 👍 

Thanks @willcohen

**Comment by normanmaurer** [comment]
Actually no... It seems like the latest AdoptJDK 8 release is still missing @jponge

**Comment by jponge** [comment]
Oops, I thought it was fixed because I saw another flavour being up-to-date again.

**Comment by normanmaurer** [comment]
Any updates ?

**Comment by trustin** [comment]
I also found AdoptOpenJDK 15.0.2 is not there. At the moment I've switched to asdf-vm which uses the metadata from https://github.com/joschi/java-metadata by @joschi.

**Comment by trustin** [comment]
I didn't try, but I think you can clone `asdf-vm` repository at branch v0.8.0 and run `bin/asdf`:

```
~/.asdf/bin/asdf plugin add java
~/.asdf/bin/asdf install java adoptopenjdk-8.0.282+8
export JAVA_HOME="$HOME/.asdf/installs/java/adoptopenjdk-8.0.282+8"
export PATH="$JAVA_HOME/bin:$PATH"
```

**Comment by sazzer** [comment]
Java 16 is also missing from the list :(

**Comment by willcohen** [comment]
Fixed by #790!
